R/get_advanced_match_stats.R

```diff
@@ -147,7 +147,12 @@ fb_advanced_match_stats <- function(match_url, stat_type, team_or_player, time_p
           }
 
 
-          stat_df_output <- cbind(match_report, stat_df_output)
+          stat_df_output <- dplyr::bind_cols(match_report, stat_df_output)
```

Comment:
`dplyr::bind_cols()` allows us to not fail at this point if there's missing data for matches